packages/camera/camera/ios/Classes/FLTCam_Test.h

```diff
@@ -17,9 +17,25 @@
 /// A dictionary to retain all in-progress FLTSavePhotoDelegates. The key of the dictionary is the
 /// AVCapturePhotoSettings's uniqueID for each photo capture operation, and the value is the
 /// FLTSavePhotoDelegate that handles the result of each photo capture operation. Note that photo
-/// capture operations may overlap, so we have to keep track of multiple delegates in progress,
+/// capture operations may overlap, so FLTCam has to keep track of multiple delegates in progress,
 /// instead of just a single delegate reference.
 @property(readonly, nonatomic)
     NSMutableDictionary<NSNumber *, FLTSavePhotoDelegate *> *inProgressSavePhotoDelegates;
 
+/// Delegate callback when receiving a new video or audio sample.
+/// Exposed for unit tests.
+- (void)captureOutput:(AVCaptureOutput *)output
+    didOutputSampleBuffer:(CMSampleBufferRef)sampleBuffer
+           fromConnection:(AVCaptureConnection *)connection;
+
+/// Init function exposed for unit test.
+/// `captureSession` is injected to avoid mocking `alloc` function.
```

Comment:
Please remove this; there's no reason to justify a common practice (allowing DI via initializers) in contrast to a rare practice (mocking alloc, which I don't think I've ever seen other than this plugin).